client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-list-renderer.tsx

```diff
@@ -35,11 +35,21 @@ const PatternListItem = ( {
 	isShown,
 	onSelect,
 }: PatternListItemProps ) => {
-	const [ inViewOnce, setInViewOnce ] = useState( false );
-	const ref = useInView< HTMLButtonElement >( () => setInViewOnce( true ), {
-		threshold: [ 0 ],
+	const ref = useRef< HTMLButtonElement >();
+	const { ref: inViewRef, inView: inViewOnce } = useInView( {
+		triggerOnce: true,
 	} );
 
+	const setRefs = useCallback(
+		( node?: Element | null | undefined ) => {
+			if ( node ) {
+				ref.current;
```

Comment:
we have to assign the node to `ref.current`, right?

Reply:
I think you are right. I think it should be something like this:

```jsx
const setRefs = useCallback(
  ( node?: Element | null | undefined ) => {
    if ( node ) {
      ref.current = node as HTMLButtonElement;
      inViewRef( node );
    }
  },
  [ inViewRef ]
);
```

Reply:
Here is a [PR](https://github.com/Automattic/wp-calypso/pull/75613) for that.